frappe/core/doctype/view_log/view_log.py

```diff
@@ -0,0 +1,10 @@
+# -*- coding: utf-8 -*-
+# Copyright (c) 2018, Frappe Technologies and contributors
+# For license information, please see license.txt
+
+from __future__ import unicode_literals
+import frappe
+from frappe.model.document import Document
+
+class Viewlog(Document):
+	pass
```

Comment:
create a joint index on `reference_type`, `reference_name` since this table will get very large very quickly